File: 프로그래머스/unrated/134240. 푸드 파이트 대회/푸드 파이트 대회.java
class Solution {
    public String solution(int[] food) {
        StringBuilder left_Line = new StringBuilder();
        StringBuilder right_Line = new StringBuilder();
        for (int i = 1; i < food.length; i++) {
            if (food[i] > 1) {
                left_Line.append(String.valueOf(i).repeat(food[i]/2));
                right_Line.append(String.valueOf(i).repeat(food[i]/2));
            }
        }
        left_Line.append(0);
        return left_Line.toString() + right_Line.reverse();
    }
}

File: 프로그래머스/unrated/134240. 푸드 파이트 대회/푸드 파이트 대회.py
def solution(food):
    foodL = []
    foodR = []
    for i in range(1,len(food)):
        if food[i] > 1:
            foodL.append(str(i)*(food[i]//2))
            foodR.append(str(i)*(food[i]//2))
    foodL.append(0)
    result = foodL+sorted(foodR,reverse=True)
    return ''.join(map(str, result))